src/one_dragon/base/controller/pc_controller_base.py

```
@@ -255,16 +261,47 @@ def get_mouse_sensitivity():
     return speed.value
 
 
-def drag_mouse(start: Point, end: Point, duration: float = 0.5):
+def drag_mouse(start: Point, end: Point, duration: float = 0.5, steps: int = 20):
     """
     按住鼠标左键进行画面拖动
     :param start: 原位置
     :param end: 拖动位置
     :param duration: 拖动鼠标到目标位置，持续秒数
```

Ruff (0.12.2)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)